charts/kargo/templates/api/configmap.yaml

```diff
@@ -18,16 +18,16 @@ data:
   TLS_CERT_PATH: /etc/kargo/tls.crt
   TLS_KEY_PATH: /etc/kargo/tls.key
   {{- end }}
-  PERMISSIVE_CORS_POLICY_ENABLED: {{ quote .Values.api.enablePermissiveCORSPolicy }}
+  PERMISSIVE_CORS_POLICY_ENABLED: {{ default "false" (quote .Values.api.enablePermissiveCORSPolicy) }}
   {{- if .Values.api.adminAccount.enabled }}
   ADMIN_ACCOUNT_ENABLED: "true"
   {{- if or .Values.api.tls.enabled (and .Values.api.ingress.enabled .Values.api.ingress.tls.enabled) }}
   ADMIN_ACCOUNT_TOKEN_ISSUER: https://{{ .Values.api.host }}
   {{- else }}
   ADMIN_ACCOUNT_TOKEN_ISSUER: http://{{ .Values.api.host }}
   {{- end }}
-  ADMIN_ACCOUNT_TOKEN_AUDIENCE: {{ .Values.api.host }}
-  ADMIN_ACCOUNT_TOKEN_TTL: {{ .Values.api.adminAccount.tokenTTL }}
+  ADMIN_ACCOUNT_TOKEN_AUDIENCE: {{ quote .Values.api.host }}
+  ADMIN_ACCOUNT_TOKEN_TTL: {{ quote .Values.api.adminAccount.tokenTTL }}
```

Comment:
Either of these could reasonably have been left undefined by a user. Go code will catch the issue.

Quotes added to avoid https://github.com/argoproj/argo-cd/issues/15566.